src/EcomDestinationWriter.cs

```diff
@@ -715,6 +716,20 @@ private int LastDetailId
         set { _lastDetailId = value; }
     }
 
+    private int _lastStockLocationGroupId = -1;
```

Comment:
This seems to be a bit redundant, this private variable is never actually used by the looks of it, but instead "LastStockLocationGroupId" is always used, as such I would propose we just make a simple method for this instead of baking it into a getter and setter of aother private property